### PR TITLE
added membership card page

### PR DESCRIPTION
**Pull request by wyland1**

### CODE REVIEW

File: src/pages/membership.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                        this wonderful journey with us!
+                        </Text>
+                    <Button 
+                        href="https://docs.google.com/forms/d/e/1FAIpQLSfgdUDFC9smFFwF-4DvmULrPQSuNbf2nHQLZiskcDazZaW0WQ/viewform"
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        variant="contained"
+                        color="secondary"
+                        >
+                            Form
```
Comment (ameliahsu) [inline src/pages/membership.tsx]: can we change this to "order now"

File: src/pages/membership.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                </Section>
+
+                <Section title="Pricing">
+                    <Text>$15 for 1 card</Text>
+                    <Text>$27 for 2 cards</Text>
+                    <Text>$48 for 4 cards</Text>
+                    <Text>$100 for 10 cards</Text>
+                </Section>
+
+                <Section title="Form">
```
Comment (ameliahsu) [inline src/pages/membership.tsx:99]: perhaps change this to "Order Your Membership Card Today!" or something like that

File: src/pages/membership.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                        A TASA membership card brings fun, happiness, 
+                        and premium benefits right to your door. These 
+                        benefits include 10% off Cuppa Tea, Class520, and Aroy 
+                        Thai (more to come!!). In addition, this card 
+                        will grant you free event tickets and raffle 
+                        tickets at our major events (Tour of Taiwan and 
+                        Night Market)! For all of these perks, you only 
+                        have to pay a small price of $15 or bundle with 
+                        some friends by buying a group package!
+                    </Text>
```
Comment (ameliahsu) [inline src/pages/membership.tsx:59]: Can we change the last 2 sentences to
"In addition, this card will award you free event tickets AND raffle tickets at two of our major events: Tour of Taiwan and Night Market! Receive all of these perks for the small cost of $15, or get some friends together and buy a group bundle!"

**Review by ameliahsu — CHANGES_REQUESTED** [review]

**Review by wyland1 — COMMENTED** [review]
updated w/ suggestions

**Pull request merged**